modules/serverless-logic-supporting-services-eventing-system-config.adoc

```diff
@@ -0,0 +1,252 @@
+
+[NOTE]
+====
+In production environments, use a production-ready broker, such as the Knative Kafka Broker, for enhanced scalability and reliability.
+====
+
+The following example displays the Job Service eventing system configuration:
+
+.Example of Job Service eventing system configuration 
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform-example
+spec:
+  services:
+    jobService:
+      source:
+        ref:
+          name: jobs-service-source-example-broker <1>
+          namespace: jobs-service-source-example-broker-namespace <2>
+          apiVersion: eventing.knative.dev/v1
+          kind: Broker
+      sink:
+        ref:
+          name: jobs-service-sink-example-broker <3>
+          namespace: jobs-service-sink-example-broker-namespace <4>
+          apiVersion: eventing.knative.dev/v1
+          kind: Broker
+----
+
+<1> Specifies the Knative Eventing Broker from which the Job Service consumes events.
+<2> Optional: Defines the namespace of the Knative Eventing Broker. By default, this is the SonataFlowPlatform namespace. It is recommended to create the Knative Eventing Broker in the same namespace as the SonataFlowPlatform.
+<3> Specifies the Knative Eventing Broker on which the Job Service produces events.
+<4> Optional: Defines the namespace of the Knative Eventing Broker. By default, this is the SonataFlowPlatform namespace. It is recommended to create the Knative Eventing Broker in the same namespace as the SonataFlowPlatform.
+
+[id="serverless-logic-supporting-services-cluster-scoped-eventing-system-config_{context}"]
+== Cluster-scoped eventing system configuration for Supporting Services
+
+When you deploy cluster-scoped supporting services, the supporting services automatically link to the Broker specified in the `SonataFlowPlatform` CR, which is referenced by the `SonataFlowClusterPlatform` CR.
+
+[id="serverless-logic-supporting-services-eventing-system-precedence-rule_{context}"]
+== Eventing system configuration precedence rules for Supporting Services
+
+The {ServerlessLogicOperatorName} follows a defined order of precedence to configure the eventing system for a supporting service. 
+
+The rules are as follows:
+
+1. If the supporting service has its own eventing system configuration, using either the Data Index eventing system or Job Service eventing system configuration, that configuration takes precedence.
+
+2. If the `SonataFlowPlatform` CR enclosing the supporting service is configured with a Platform-scoped eventing system, that configuration takes precedence.
```

Comment:
SonataFlowPlatform is highlighted here so it should be highlighted elsewhere?

Reply:
Platform-scoped > platform-scoped

Reply:
Done!